meson.build

```diff
@@ -6,7 +6,7 @@ project(
         ['sh', 'build/general_variables_for_meson.sh', 'version'],
         check : true).stdout(),
     license : 'Apache-2.0',
-    meson_version : '>= 0.47.0',
+    meson_version : '>= 1.0.0',
```

Comment:
Meson version required version bumped due to `'in' operator for string variables`. Is it worth its backward compatibility breaking? I think yes. except if a better solution found.

Those are all of my points about this PR.

Reply:
But you should update documentation accordingly